new/PRS14584GabraSe.xml

```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14584GabraSe" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gabra Śǝllāse</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2024-11-07">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ገብረ ሥላሴ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra Śǝllāse</persName>
+                    <faith type="EOTC"/>
```

Comment:
Do you have positive evidence for his faith (same for the others)?

Reply:
Only the fact that they are mentioned in a Maṣḥafa gǝnzat.
Do you think it is better to leave this information out?

Reply:
I only include information about faith if there are reasons to be certain, but I honestly don't know if this is the best practice :) how do you do it, @CarstenHoffmannMarburg ?

Reply:
I am rather cautious and usually do not give faith unless there is a clear reason, for example if a person has a position in the church hierarchy. But other encoders in our team use EOTC more often. I think that the possession of a burial book or other religious literature is an indicator of the faith of the owner, and therefore it is legitimate to keep EOTC in the record here, although I think it may also be a bit ambiguous in principle, as people may own books for reasons other than their intended use in service, for example as a valuable or prestigious item, or for personal study. It is up to you to decide whether this is enough to code a person as an EOTC.

Reply:
Better to leave this information out, then. There are no indications that these persons belong to an ecclesiastical hierachy.